### PR TITLE
#209 update cpplint config from dynawo

### PR DESCRIPTION
**Pull request by zamarrenolm**

### CODE REVIEW

File: CPPLINT.cfg [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-root=sources
+extensions=cpp,h,hpp,hxx
+root=dynawo/sources
 linelength=160
 
 # Filter checks
-filter=-legal/copyright
+filter=-build/c++11
 filter=-build/include
+filter=-build/c++11
```
Comment (lecourtoisflo) [inline CPPLINT.cfg:10]: duplicated filter

File: CPPLINT.cfg [hunk 2]
```diff
@@ -1,13 +1,14 @@
 set noparent
 headers=h,hpp,hxx
-root=sources
+extensions=cpp,h,hpp,hxx
+root=dynawo/sources
```
Comment (lecourtoisflo) [inline CPPLINT.cfg:4]: dynawo/sources directory doesn't exist in DFL

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

File: CPPLINT.cfg (hunk 2 above)
Comment (zamarrenolm) [reply CPPLINT.cfg]: Fixed. The cpplint analysis still was performed because the cpplint wrapper is invoked with the proper source directory, and is responsible for building the list of files.

File: CPPLINT.cfg (hunk 1 above)
Comment (zamarrenolm) [reply CPPLINT.cfg]: Fixed. It is also duplicated in dynawo.

File: CPPLINT.cfg [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 # Filter checks
-filter=-legal/copyright
+filter=-build/c++11
 filter=-build/include
 filter=-build/include_what_you_use
 filter=-build/include_order
-filter=-build/c++11
 filter=-runtime/references
-filter=-runtime/string
+filter=-runtime/int
```
Comment (lecourtoisflo) [inline CPPLINT.cfg:13]: Why did you add this one ? verboten C basic types should not be used

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

File: CPPLINT.cfg (hunk 3 above)
Comment (zamarrenolm) [reply CPPLINT.cfg:13]: It comes from the configuration used in dynawo. I just copied the current CPPLINT.cfg found in master branch of dynawo. Should this filter be removed here and also in dynawo ?

Comment (lecourtoisflo) [reply CPPLINT.cfg:13]: In dynawo, it shouldn't. But in DFL, maybe it should because we have less constraints. @gautierbureau what do you think ?

Comment (lecourtoisflo) [reply CPPLINT.cfg:13]: After looking at the definitions of the filters, runtime/string and runtime/int should be present in cpplint of DFL

Comment (zamarrenolm) [reply CPPLINT.cfg:14]: Done, the line for runtime/string was recovered

Comment (lecourtoisflo) [reply CPPLINT.cfg:14]: No it is the opposide, the filters should be checked. Your lines removed the check

Comment (zamarrenolm) [reply CPPLINT.cfg]: Sorry for the confusion. I updated CPPLINT.cfg so both filters are not discarded.

**Review by lecourtoisflo — APPROVED** [review]

**Review by marcochiaramello — APPROVED** [review]
